test/e2e/resource_test.go

```diff
@@ -198,8 +198,13 @@ var _ = ginkgo.Describe("[resource-status collection] resource status collection
 					latestSvc, err := kubeClient.CoreV1().Services(serviceNamespace).Get(context.TODO(), serviceName, metav1.GetOptions{})
 					g.Expect(err).NotTo(gomega.HaveOccurred())
 
+					for i := range latestSvc.Status.LoadBalancer.Ingress {
+						if latestSvc.Status.LoadBalancer.Ingress[i].IPMode != nil {
+							latestSvc.Status.LoadBalancer.Ingress[i].IPMode = nil
```

Comment:
Will the code need to be adjusted if all our running versions support this field in the future? Maybe add some notes here.